src/Home.js

```diff
@@ -1,17 +1,111 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import React, { useState, useEffect } from 'react';
+import { StyleSheet, Text, View, FlatList, Pressable } from 'react-native';
+import { DataStore } from 'aws-amplify';
+import { Todo } from './models';
 
 export default function Home() {
+
+    const [todos, setTodos] = useState([]);
+
+    let subscription;
+
+    useEffect(() => {
+        onQuery = async () => {
+            const todos_list = await DataStore.query(Todo);
+            setTodos(todos_list);
+        };
+        onQuery();
+        if (typeof subscription === 'undefined') {
+            subscription = DataStore.observe(Todo).subscribe((msg) => {
+                onQuery();
+            });
+        }
+
+        return function cleanup() {
+            subscription.unsubscribe();
+        }
+    }, []);
+
+    async function setComplete(updateValue, todo) {
+        await DataStore.save(
+            Todo.copyOf(todo, updated => {
+                updated.isComplete = updateValue
+            })
+        );
+    }
+
+    const todoItem = ({ item }) => (
+        <View style={styles.todoContainer}>
+            <View>
+                <Text style={styles.todoHeading}> {item.name} </Text>
+                <Text style={styles.todoDescription}> {item.description} </Text>
+            </View>
+            <Pressable
+                style={[styles.checkbox, item.isComplete && styles.selectedCheckbox]}
+                onPress={() => {
+                    setComplete(!item.isComplete, item)
+                }}
+            >
+                <Text style={[styles.checkboxText, item.isComplete && styles.selectedCheckboxText]}> {(item.isComplete) ? "✓" : ""} </Text>
```

Comment:
Can we remove the white space inside the content of these `Text` tags?

Reply:
Got it. Removed the whitespace.